src/MeetingsSDKAdapter.js

```diff
@@ -41,8 +40,8 @@ import {
 const EVENT_MEDIA_READY = 'media:ready';
 const EVENT_MEDIA_STOPPED = 'media:stopped';
 const EVENT_STATE_CHANGE = 'meeting:stateChange';
-const EVENT_LOCAL_SHARE_STOP = 'meeting:stoppedSharingLocal';
-const EVENT_LOCAL_SHARE_START = 'meeting:startedSharingLocal';
+const EVENT_LOCAL_SHARE_STOPPED = 'meeting:stoppedSharingLocal';
+const EVENT_LOCAL_SHARE_STARTED = 'meeting:startedSharingLocal';
 const EVENT_REMOTE_SHARE_START = 'meeting:startedSharingRemote';
```

Comment:
seems that these event names also need to be updated. Maybe we can create a different task named "Fix SDK meeting event names used in adapter" just for that